wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -610,7 +610,7 @@ extension ClientListViewController: UserObserving {
 private extension UserClient {
 
     var resolvedMLSThumbprint: String? {
-        e2eIdentityCertificate?.mlsThumbprint ?? mlsPublicKeys.ed25519
+        e2eIdentityCertificate?.mlsThumbprint
```

Comment:
from my understanding this was wrong - does the change affect anything else connected too this view controller?

Reply:
I don't think.
do you think we should keep `resolvedMLSThumbprint`?

Reply:
Thanks, done!